internal/biz/relationships.go

```diff
@@ -41,6 +43,18 @@ func (rc *CheckUsecase) Check(ctx context.Context, check *v0.CheckRequest) (*v0.
 	return rc.repo.Check(ctx, check)
 }
 
+type IsBackendAvaliableUsecase struct {
```

Comment:
These guys make sense but probably belong in their own file. (..Recognizing that this file's already a little busy with things that maybe don't belong here)

Reply:
Moved the types and corresponding funcs `NewIsBackendAvailableUsecase` and `IsBackendAvailable` to `health.go`